src/components/markdown_editor/markdown_editor.spec.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/// <reference types="../../../cypress/support"/>
+
+import React from 'react';
+import { EuiMarkdownEditor } from './index';
+
+describe('EuiMarkdownEditor', () => {
+  describe('toolbar interactions', () => {
+    it('keeps focus on the input box when clicking a disabled item', () => {
+      const value = `**bold**
+
+_italic_
+
+> quote
+
+\`inline codeblock\`
+
+[link](https://elastic.co)
+
+!{tooltip[text](help)}`;
+      cy.realMount(
```

Comment:
super minor newline nit, this tripped me up a bit at first glance
```suggestion
!{tooltip[text](help)}`;

      cy.realMount(
```

Alternatively I'd also say you could put `value` up on line 13 before the first describe() block if you don't want to deal with indentation shenanigans

Reply:
added the newline 👍